Cython/Compiler/ExprNodes.py

```diff
@@ -500,18 +500,23 @@ def result(self):
         else:
             return self.calculate_result_code()
 
+    def _make_move_result(self, result):
```

Comment:
Sorry for introducing yet another `_make_move*` result into the mix, but I don't understand the naming here.

This was how I implemented the approach @scoder mentions in https://github.com/cython/cython/pull/3821#issuecomment-693925316.

Open to other suggestions here